superchain/extra/addresses/sepolia-dev-0/base-devnet-0.json

```diff
@@ -0,0 +1,10 @@
+{
+  "AddressManager": "0x882a60911d00867Fe4ea632C479cc48e583A8D69",
+  "L1CrossDomainMessengerProxy": "0x2cbD403d5BA3949D24ee4dF57805eaC612C2662f",
+  "L1ERC721BridgeProxy": "0xc3016ED03E087d092d576B585F5222fFD9cadc10",
+  "L1StandardBridgeProxy": "0x5638e55db5Fcf7A58df525F1098E8569C8DbA80c",
+  "L2OutputOracleProxy": "0xB5901509329307E3f910f333Fa9C4B4A8EE7CE1A",
+  "OptimismMintableERC20FactoryProxy": "0xEAa11178375e6B1078d815d6F9F85cBbb69b09Cd",
+  "OptimismPortalProxy": "0x579c82A835B884336B632eeBeCC78FA08D3291Ec",
+  "ProxyAdmin": "0xC5aE9023bFA79124ffA50169E1423E733D0166f1"
+}
```

Comment:
Apologies, since this is an oversight on our end: would you mind adding the address for `SystemConfigProxy`? We need to validate that contract too.

Reply:
Added this in https://github.com/ethereum-optimism/superchain-registry/pull/67/commits/398632095c29e3e25f3bf11b0f68907f324ca0a0

Reply:
Many thanks. I ran the "in flight" validation code against this PR:
```
=== RUN   TestContractVersions
    superchain-version_test.go:86: sepolia-dev-0/Base devnet 0.L1CrossDomainMessengerProxy.version=1.6.0 (acceptable compared to 1.6.0)
    superchain-version_test.go:86: sepolia-dev-0/Base devnet 0.L1ERC721BridgeProxy.version=1.3.0 (acceptable compared to 1.3.0)
    superchain-version_test.go:86: sepolia-dev-0/Base devnet 0.L1StandardBridgeProxy.version=1.4.0 (acceptable compared to 1.4.0)
    superchain-version_test.go:86: sepolia-dev-0/Base devnet 0.L2OutputOrcaleProxy.version=1.5.0 (acceptable compared to 1.5.0)
    superchain-version_test.go:86: sepolia-dev-0/Base devnet 0.OptimismMintableERC20FactoryProxy.version=1.4.0 (acceptable compared to 1.4.0)
    superchain-version_test.go:86: sepolia-dev-0/Base devnet 0.OptimismPortalProxy.version=1.9.0 (acceptable compared to 1.9.0)
    superchain-version_test.go:86: sepolia-dev-0/Base devnet 0.SystemConfigProxy.version=1.7.0 (acceptable compared to 1.7.0)
--- PASS: TestContractVersions (0.40s)
PASS
ok  	github.com/ethereum-optimism/superchain-registry/validation	0.624s
```